src/queue.js

```diff
@@ -8,14 +8,14 @@ export class Node {
 
 export default class Queue {
   constructor() {
-    this.length = 0
+    this.size = 0
     this.head = null
     this.tail = null
   }
 
   enqueue(value) {
     const node = new Node(value)
-      if (this.length) {
+      if (this.size) {
```

Comment:
I like to be explicit in my boolean expressions to avoid possible bugs, and to make it clear in the code what my expectation is:
```js
if( this.size === 0 ) {

} else {

}
```
Having said that, I see repetition here.  What we are really asking is, "Is this Queue empty"?  Consider using the method we've defined for that question:
```js
if( this.empty() ) {

} else {

}
```